includes/utils/class-amp-dom-utils.php

```diff
@@ -123,6 +135,14 @@ function( $matches ) {
 			return false;
 		}
 
+		/*
+		 * Remove pre-HTML5-style encoding declaration if added above.
+		 */
+		$meta_http_equiv_element = $dom->getElementById( 'meta-http-equiv' );
```

Comment:
This could be made a tiny more performant by first checking `version_compare( LIBXML_DOTTED_VERSION, '2.8', '<' )`. The result of the first call could be put inside a variable to re-use here.

Reply:
In fact, you can see there is another such check for the libxml version on line 99. So all of these checks could re-use this same var, e.g.:

```php
$is_old_libxml = version_compare( LIBXML_DOTTED_VERSION, '2.8', '<' );
```

Reply:
Okay, wrapping this in a version check makes sense to me, as does introducing a variable. I'd be inclined to name the variable something more specific, like `$is_pre_2_8_libxml`, but if you have a specific preference for the name I'll go with that.

Reply:
Either is fine with me. Since we're only testing for one version of libxml currently, the a simple “old“ name should suffice.